src/style.css

```diff
@@ -1,3 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it to avoid overflow or accidental scroll bar. If a fixed height is not a requirement, consider using min-height or padding instead.